BTrees/tests/test_OOBTree.py

```diff
@@ -352,3 +391,6 @@ def test_suite():
         unittest.makeSuite(OOTreeSetPyConflictTests),
         unittest.makeSuite(OOModuleTest),
     ))
+
+if __name__ == '__main__':
+    unittest.main(defaultTest='test_suite')
```

Comment:
While I sympathize with the motivation, we really need to avoid adding scripty baggage to the unit test modules.